src/components/Movie.js

```diff
@@ -1,45 +1,60 @@
 import React from "react";
 import "./Movie.css";
-// import ""
+import { useEffect, useState } from "react";

```

Comment:
You can include it in 1 row.

Reply:
BE sure that you remove all related folder and files aspecially when you change logic
. Like /componenets/img